.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true,
+    "python.formatting.provider": "black"
+}
```

Comment:
IMHO, this should be gitignored

Reply:
fair, I have ragequitted vscode in favor of nvim since then